### DISCUSSION TITLE
Is it possible to get autoshift on ,./ <>? ?

### DISCUSSION THREAD

**Discussion by abandon42** [manna-harbour/miryoku#101] [Q&A] [answered]
It would be most helpful for ?

**Comment by manna-harbour** [comment, marked as answer]
It's enabled for all non-alphas, but auto shift isn't compatible with tap-hold.

If you don't need AltGr or Button, you can remove them from Base with a custom layer. See https://github.com/manna-harbour/miryoku/discussions/85. 

Alternatively, enable retro shift. See https://docs.qmk.fm/#/feature_auto_shift?id=retro-shift.
